CHANGELOG.md

```diff
@@ -2,12 +2,24 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### Unreleased
+
+* Add syntax highlighting for struct method import expression, fixes [#566](https://github.com/a-h/templ/issues/566) in #38
+
 ### [0.0.21](https://github.com/a-h/templ-vscode/compare/v0.0.20...v0.0.21) (2024-02-27)
 
 ### Features
 
 * Add restart command
 
+### [0.0.20](https://github.com/templ-go/templ-vscode/compare/v0.0.19...v0.0.20) (2024-02-18)
+
+* Resolve single line import expression syntax highlighting, fixes [#508](https://github.com/a-h/templ/issues/508) in #29
+* Refactor import-expression syntax regex matching, fixes [#524](https://github.com/a-h/templ/issues/524) in #30
+* Refactor if/else-if statement syntax regex matching, fixes [#524](https://github.com/a-h/templ/issues/524) in #31
+* Add syntax highlighting for style/script tags, fixes [#489](https://github.com/a-h/templ/issues/489) in #32
+* Add syntax highlighting support for HTML attribute conditional statements, fixes [#533](https://github.com/a-h/templ/issues/533) in #33
```

Comment:
I hadn't added changelog entries in my previous PRs, so wanted get a backlog for them on the previous release.